### DISCUSSION TITLE
Docker excec import - copy don't move

### DISCUSSION THREAD

**Discussion by BumbleCast** [photoprism/photoprism#2818] [Q&A] [answered]
Hi all,
i'd like to trigger a library import withotu moving the files (in the webgui it would be Settings - Import ('Move files' unchecked) - Import).
I found the command 
docker exec -ti photoprism photoprism import
https://docs.photoprism.app/getting-started/docker/

However, the command moves the files and I don't find any info about a potential flag or option to just copy the files. The --help command also doesn't give additional info.
Any idea? Thanks!

**Comment by lastzero** [comment, marked as answer]
The --help command also doesn't give additional info... :eyes: 

```
photoprism --help
NAME:
   photoprism - PhotoPrism® CE

USAGE:
   photoprism [global options] command [command options] [arguments...]

VERSION:
   221026-90e5283a-Linux-AMD64-DEBUG

DESCRIPTION:
   PhotoPrism® is an AI-Powered Photos App for the Decentralized Web

COMMANDS:
   start, up    Starts the Web server
   stop, down   Stops the Web server in daemon mode
   status       Checks if the Web server is running
   index        Indexes original media files
   mv, import   Moves media files to originals
   cp, copy     Copies media files to originals
```

**Reply by lastzero** [reply]
👉 `cp, copy     Copies media files to originals`

**Comment by BumbleCast** [comment]
Ah, my bad.
I run docker exec -ti photoprism photoprism import --help and it only gave info about the mv command.
Thanks!
